app/lua/lnodemcu.h

```diff
@@ -38,9 +40,9 @@
 #define LROT_ENTRYREF(rt)    (rt ##_entries)
 #define LROT_TABLEREF(rt)    (&rt ##_ROTable)
 #define LROT_BEGIN(rt,mt,f)  LROT_TABLE(rt); \
-  static const ROTable_entry rt ## _entries[] = {
+  static ROTable_entry rt ## _entries[] = {
 #define LROT_ENTRIES_IN_SECTION(rt,s) \
-  static const ROTable_entry LOCK_IN_SECTION(s) rt ## _entries[] = {
+  static ROTable_entry LOCK_IN_SECTION(s) rt ## _entries[] = {
```

Comment:
'cos the `ROTable_entry` typedef is already a `const struct`.

Reply:
Note that whereas ` ROTable_entry` vectors are always RO, this is one `ROTable` declaration where this isn't the case and that is for the `ROM` `ROTable`.  This is because whilst the `const` vector is collated using linker magic, its size can't easily be determined at link time, and so `lua_createrotable()` is used during Lua initialisation to build this one `ROTable` struct in RAM which points to the  ROTable_entry` vector in flash.